hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/S3AFileSystem.java

```diff
@@ -503,6 +501,15 @@ private void doBucketProbing() throws IOException {
     }
   }
 
+  private void logDnsLookup() {
```

Comment:
Let's put this in NetworkBinding, so that we can keep S3AFileSystem less bloated than it already is.